src/schema/schema.graphql

```diff
@@ -1280,6 +1280,11 @@ type Contact {
 	members: [ContactListMember]
 }
 
+type otherAttribute{
```

Comment:
Capitalize ? Can be renamed to OtherContactAttribute

Reply:
Done.